go.mod

```diff
@@ -7,6 +7,7 @@ require (
 	github.com/ghodss/yaml v1.0.0
 	github.com/gin-contrib/cors v1.3.1
 	github.com/gin-gonic/gin v1.7.4
+	github.com/golang-jwt/jwt v3.2.2+incompatible
```

Comment:
The https://github.com/golang-jwt/jwt package is at version v4.1.0. Any particular reason for sticking with v3?

They did the Go-module compatible thing and changed the path to `github.com/golang-jwt/jwt/v4` (https://github.com/golang-jwt/jwt/blob/main/MIGRATION_GUIDE.md)